src/System.Text.Encodings.Web/src/System.Text.Encodings.Web.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <RootNamespace>System.Text.Encodings.Web</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release;netstandard2.1-Debug;netstandard2.1-Release</Configurations>
+    <Configurations>netcoreapp3.0-Debug;netcoreapp3.0-Release;netstandard-Debug;netstandard-Release;netstandard2.1-Debug;netstandard2.1-Release</Configurations>
```

Comment:
Continuing to use `TargetsNetCoreApp` within item group conditions is fine, even with this change, correct?

https://github.com/dotnet/corefx/blob/3b6ec2ba944fb72f921f95b99b6ebf2e7bb3dc5a/Directory.Build.props#L149

What about the using the SDK constant in source for if/defs? Should these be changed to TFM specific (i.e. `NETCOREAPP3_0`)? I don't see any other use of that in the repo, but maybe this is the first instance. I see older TFMS though.
https://github.com/dotnet/corefx/blob/bdd33976fe3713cc3e78b83cf2a1176c70b793be/src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/TextEncoder.cs#L13-L16

Reply:
Yes using TargetsNetCoreApp is fine.  On the defines: in master I believe we use the SDK pattern where netcoreapp means any version, so these should be fine.  Double check the IL to be sure (or test for the difference).

Reply:
> Double check the IL to be sure (or test for the difference).

Yep, that's what I am doing to verify. I could instead use a custom constant, like we do in S.T.Json (and what I end up doing for release/3.1).
https://github.com/dotnet/corefx/blob/bdd33976fe3713cc3e78b83cf2a1176c70b793be/src/System.Text.Json/src/System.Text.Json.csproj#L9